tests/integration/annotation_import/conftest.py

```diff
@@ -493,29 +502,37 @@ def configured_project(client, ontology, rand_gen, image_url):
         data_row_ids=data_row_ids,
         wait_processing_max_seconds=DATA_ROW_PROCESSING_WAIT_TIMEOUT_SECONDS,
         sleep_interval=DATA_ROW_PROCESSING_WAIT_SLEEP_INTERNAL_SECONDS)
-    project.datasets.connect(dataset)
+    project.create_batch(
+        rand_gen(str),
+        data_row_ids,  # sample of data row objects
+        5  # priority between 1(Highest) - 5(lowest)
+    )
     project.data_row_ids = data_row_ids
     yield project
     project.delete()
-    dataset.delete()
 
 
 @pytest.fixture
 def configured_project_pdf(client, ontology, rand_gen, pdf_url):
     project = client.create_project(name=rand_gen(str),
-                                    queue_mode=QueueMode.Dataset)
+                                    queue_mode=QueueMode.Batch,
+                                    media_type=MediaType.Pdf)
     dataset = client.create_dataset(name=rand_gen(str))
     editor = list(
         client.get_labeling_frontends(
             where=LabelingFrontend.name == "editor"))[0]
     project.setup(editor, ontology)
-    data_row_ids = []
-    data_row_ids.append(dataset.create_data_row(pdf_url).uid)
+    data_row = dataset.create_data_row(pdf_url)
+    data_row_ids = [data_row.uid]
     project._wait_until_data_rows_are_processed(
         data_row_ids=data_row_ids,
         wait_processing_max_seconds=DATA_ROW_PROCESSING_WAIT_TIMEOUT_SECONDS,
         sleep_interval=DATA_ROW_PROCESSING_WAIT_SLEEP_INTERNAL_SECONDS)
```

Comment:
the below create_batch function calls _wait_until_data_rows_are_processed internally, so no need for this here